executor/aggregate.go

```diff
@@ -512,17 +513,20 @@ func (w *HashAggFinalWorker) run(ctx sessionctx.Context, waitGroup *sync.WaitGro
 	w.getFinalResult(ctx)
 }
 
-// Next implements the Executor Next interface.
-func (e *HashAggExec) Next(ctx context.Context, req *chunk.RecordBatch) error {
+func (e *HashAggExec) Next(ctx context.Context, req *chunk.Chunk) error {
+	if span := opentracing.SpanFromContext(ctx); span != nil && span.Tracer() != nil {
```

Comment:
can be removed? see https://github.com/pingcap/tidb/pull/10875

Reply:
My mistake.
The unnecessary code has been removed.